plugins/datadog/app/models/datadog_notification.rb

```diff
@@ -23,7 +24,7 @@ def deliver
     )
 
     client = Dogapi::Client.new(api_key, nil, "")
-    status, _ = client.emit_event(event)
+    status, = client.emit_event(event)
```

Comment:
this is weird syntax, is it equivalent to `status = client.emit_event(event)[0]` ?